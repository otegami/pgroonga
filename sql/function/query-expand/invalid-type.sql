CREATE TABLE synonyms (
  term text PRIMARY KEY,
  synonym int
);

SELECT pgroonga_query_expand('synonyms', 'term', 'synonym', 'PGroonga');

DROP TABLE synonyms;
