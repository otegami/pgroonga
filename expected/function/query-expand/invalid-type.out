CREATE TABLE synonyms (
  term text PRIMARY KEY,
  synonym int
);
SELECT pgroonga_query_expand('synonyms', 'term', 'synonym', 'PGroonga');
ERROR:  pgroonga: [query-expand] synonyms column isn't text type nor text[] type: <synonyms>.<synonym>
DROP TABLE synonyms;
